DependencyInjection/TrikoderOAuth2Extension.php

```diff
@@ -64,7 +65,7 @@ private function configureAuthorizationServer(ContainerBuilder $container, array
     {
         $authorizationServer = $container
             ->getDefinition('league.oauth2.server.authorization_server')
-            ->replaceArgument('$privateKey', $config['private_key'])
+            ->replaceArgument('$privateKey', new CryptKey($config['private_key'], null, false))
```

Comment:
You were on the right track, however, when you're altering the container you should use the `Definition` and `Reference` classes to describe the changes you're applying.

```suggestion
            ->replaceArgument('$privateKey', new Definition(CryptKey::class, [
                $config['private_key'],
                null,
                false,
            ]))
```

This is why Symfony wasn't able to resolve the environment variable placeholders correctly, as seen from the failed build output:

```
LogicException: Key path "file://%env(PRIVATE_KEY_PATH)%" does not exist or is not readable
```

Reply:
Also, you forgot to apply the changes for the public key path configuration in the `configureResourceServer` method.